website/docs/cloud-docs/workspaces/organize-workspaces-with-projects.mdx

```diff
@@ -16,13 +16,13 @@ When deciding how to structure your projects, consider which groups of resources
 
 ## Permissions
 
-You must be a member of a team with the **Manage Projects & Workspaces** [organization-level permissions](/terraform/cloud-docs/users-teams-organizations/permissions#organization-permissions) permissions to create a project.
+You must be a member of a team with **Manage all projects and Manage all workspaces** [organization-level permissions](/terraform/cloud-docs/users-teams-organizations/permissions#organization-permissions) permissions to create a project.
```

Comment:
To mark that these are two separate permissions I'd recommend unbolding the "and" in the middle:
```suggestion
You must be a member of a team with **Manage all projects** and **Manage all workspaces** [organization-level permissions](/terraform/cloud-docs/users-teams-organizations/permissions#organization-permissions) permissions to create a project.
```